Cubical/Data/Sigma/Properties.agda

```diff
@@ -199,13 +100,13 @@ swapΣEquiv A B = isoToEquiv (iso (λ x → x .snd , x .fst) (λ z → z .snd ,
 
 Σ-ap-iso₁ : ∀ {ℓ} {ℓ'} {A A' : Type ℓ} {B : A' → Type ℓ'}
```

Comment:
\ell and \ell' should be variables at the top of the file

Reply:
I don't understand this name and some of the other with ap... They have nothing to do with ap?

Reply:
Any suggestion for `Σ-ap`?
It's ap in the sense that it's the action of `Σ` on paths (or isos...). In that case maybe it should be called `Σ-cong`. It doesn't seem like there's a convention for naming this sort of function yet.

Reply:
Hmm, I guess `Σ-cong` is fine then. How about not calling this lemma `Σ-ap-iso₁` but `Σ-cong-iso-fst` instead?

Reply:
Shall I change `mapˡ` and `mapʳ` to `map-fst` and `map-snd` as well?

Reply:
That would be great! Maybe we should start drafting a naming convention one of these days...